app/js/renderer.js

```diff
@@ -83,6 +83,7 @@ function updateBody(data, settings, scripts) {
   }
   $('#content').html(generatedHtml);
 
+  refreshIframe();
```

Comment:
Shouldn't the method name be plural? `refreshIframes`? Better yet, perhaps we should name it for its purpose rather than implementation. Something like `refreshMedia` or something?